pg_search/tests/pg_regress/expected/groupby_aggregate.out

```diff
@@ -31,19 +31,20 @@ WITH (
     numeric_fields='{"rating": {"fast": true}, "price": {"fast": true}}'
 );
 -- Test 1.1: Basic GROUP BY with integer field
-EXPLAIN (FORMAT TEXT, COSTS OFF, TIMING OFF)
+EXPLAIN (FORMAT TEXT, COSTS OFF, TIMING OFF, VERBOSE)
 SELECT rating, COUNT(*) AS count
 FROM products 
 WHERE description @@@ 'laptop' 
 GROUP BY rating
 ORDER BY rating;
                                                                       QUERY PLAN                                                                       
 -------------------------------------------------------------------------------------------------------------------------------------------------------
- Custom Scan (ParadeDB Aggregate Scan) on products
+ Custom Scan (ParadeDB Aggregate Scan) on public.products
+   Output: rating, now()
```

Comment:
Not something to fix here, but: we should probably replace `now()` as our placeholder with `count(..)`, or perhaps some empty-bodied functions like `paradedb.count(..)`.